composer.json

```diff
@@ -21,7 +21,10 @@
 		"composer/installers": "~1.0"
 	},
 	"require-dev": {
-		"wporg/wporg-repo-tools": "dev-trunk"
+		"phpunit/phpunit": "^9.5",
```

Comment:
I'm unsure if this is actually needed here, I suspect it's not, as it would only be needed if you're running `phpunit` outside of `wp-env`.

The below wp-phpunit & yoast deps are however required.